idb/postgres/postgres.go

```diff
@@ -706,11 +706,6 @@ func (db *IndexerDb) txnsWithNext(ctx context.Context, tx *sql.Tx, tf idb.Transa
 		return
 	}
 	tf.Limit -= uint64(count)
-	select {
```

Comment:
no reason to drop this? We _could_ rely on the QueryContext() to fail out if ctx is Done, but why not check between two biggish chunks of work?

Reply:
Right before this we pass `ctx` to `yieldTxnsThreadSimple()`, and I thought it returned an error if the context expires. Btw, is there a reason why we do not notify the user if the context expires? Seems like, he will just get incomplete data and be unaware of it.

Reply:
I returned this code, but I think we later need to improve context handling.

Reply:
I expect the most likely cause of context expiration is that HTTP client closes their connection; so the database can stop doing work when there's no client to return data to.